### DISCUSSION TITLE
Log reports

### DISCUSSION THREAD

**Discussion by MarcS1975** [docker-mailserver/docker-mailserver#3903] [Q&A]
What is the difference between Postfix log summary ("pflogsumm") and LOGWATCH_REPORT
Both variables for sender, frequency and recipient.

**Comment by casperklein** [comment]
[pflogsumm](https://linux.die.net/man/1/pflogsumm) and [logwatch](https://sourceforge.net/projects/logwatch/) are two different tools that generate log reports.

The easiest way to tell the difference would have been to activate both and look at the reports 😉

**Comment by MarcS1975** [comment]
OK, I have some curious entries in the report sent to mailserver-report@MYSERVER. 

Maybe someone can help me understand what needs fixing here. The strangest one is the warning that my **mail server** domain name does not resolve to **my IP**, which is not true. A IP reverse lookup proofs that it does resolve.
Also the TLS library problems errors I do not understand.
Any ideas?


Warnings
--------
```
master (total: 9)
         3   duplicate master.cf entry for service "policyd-spf" (private/po...
         3   duplicate master.cf entry for service "127.0.0.1:10025" ([127.0...
         3   duplicate master.cf entry for service "smtp-amavis" (private/sm...
  smtpd (total: 15)
         3   TLS library problem: error:14209102:SSL routines:tls_early_post...
         2   TLS library problem: error:14094412:SSL routines:ssl3_read_byte...
         1   TLS library problem: error:1420918C:SSL routines:tls_early_post...
         1   TLS library problem: error:142090FC:SSL routines:tls_early_post...
         1   TLS library problem: error:1417A0C1:SSL routines:tls_post_proce...
  submissions/smtpd (total: 119)
        **81   hostname MYDOMAIN does not resolve to address MYIP**
         2   TLS library problem: error:1408F10B:SSL routines:ssl3_get_recor...
         2   TLS library problem: error:1408F09C:SSL routines:ssl3_get_recor...
```

**Reply by casperklein** [reply]
You'll have to check your mail.log for each problem to get the context.

Also check if others had already [similar issues](is:issue sort:updated-desc duplicate master.cf entry for service is:closed). The first three are probably to incorrectly restarting DMS.
"TLS library problems" are often caused by spam bots.
